README.md

```diff
@@ -151,16 +152,27 @@ Write something about these
 FOLIO is designed so that different modules can be written in different
 languages with different tools.
 
-#### Java/Vert.x and Node.js
+#### Language Choice
 
-So far we have only written server-side modules in Java, using Vert.x and
-Node.js. Because we use them internally, those two technologies will have
-a prominent place in the FOLIO ecosystem and, initially, it may be easiest
-to get started using them. We provide libraries and utilities that
-help with development (especially with writing standard boiler-plate code and
-scaffolding) but we hope to eventually gain a wide coverage among other
-tools and technologies (e.g. Python, Ruby, etc.). We are counting on an active
-engagement from the community to help out in this area.
+Within FOLIO, server-side modules are primarily written in Java (with some
+in Node.js).  These are used a lot within FOLIO, therefore, we have build
+libraries and utilities to work with them (especially for standard scaffolding
+and boiler-plate code).
+
+Currently, a lot of these modules are written using Vert.x (and built with
+our [raml-module-builder](https://github.com/folio-org/raml-module-builder)
+framework).  However, this method is on the way out and being replaced
+with the 
```

Comment:
Is that statement accurate?

Reply:
I was under the impression that the current preference was for new modules (at the least) to be built with Spring Way?

Reply:
@dcrossleyau if you have language that might be better for this, we'd be happy to change it!

Reply:
I suggest avoiding comparison. RMB is one way and Spring Way is another way.

Reply:
@dcrossleyau does the new wording sound good?  We will work on the other changes shortly.

Reply:
Yes. Thanks.